crm_compassion/tests/test_crm_compassion.py

```diff
@@ -0,0 +1,282 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2015 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Albert SHENOUDA <albert.shenouda@efrei.net>
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+
+from openerp.tests import common
+from datetime import datetime
+from openerp import netsvc
+from openerp.tools import DEFAULT_SERVER_DATE_FORMAT as DF
+import logging
+import pdb
+logger = logging.getLogger(__name__)
+
+
+class test_crm_compassion(common.TransactionCase):

```

Comment:
It seems that most of your tests (recurring_contract, contract_compassion, sponsorship_compassion, ...) have the same setup method and needs. Instead of copying again and again the same code, use inheritance !

So instead of inheriting the class `common.TransactionCase` you can inherit something like `recurring_contract.test_recurring_contract`.

In that way, when you call `super` inside the setUp method, it will call the setUp method of the test in recurring_contract. You will also be able to call all private methods like `_create_contract()`, `_create_group()`, etc...

Please consider this change.